_docs/Contrib.md

```diff
@@ -1,33 +1,62 @@
-2) **WAI-ARIA state & property**: aria-prefixed markup attribute which sets the state of a component (i.e. "aria-checked" for a checkbox) or a property of it (i.e. aria-label). When combined with a role, it helps the assistive technology convey information to the user. Note that the difference between a state and property is semantic when considering the code implementation - states usually change a lot while properties do not. For a full listing see [States & Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
-3) **Implementing keyboard support**: a set of keyboard behaviors implemented for the component. For each component type, the w3 organization has provided clear instructions on how to correctly implement them ([Accordion example](https://www.w3.org/TR/wai-aria-practices/#accordion)).
-4) **Focus**: in order for an interactive component to be supported by assistive technologies, its ability to gain focus is crucial. It is important for a composite component (i.e. dropdown, radio group) to gain focus through tab and to explicitly set the tabindex on its children. See [Keyboard navigation inside components](https://www.w3.org/TR/wai-aria-practices/#kbd_general_within) for an explanation on the correct use of tabindex in composite components.
+* Read the documentation for the component in the spec. The spec will explain how to implement the following checklist for each component:
 
-### Aria-prefixed Markup Attributes
+    1) **WAI-ARIA role**: A role gives assistive technologies information about how to handle the element. There are 2 ways to assign a role to a component:
+        * Role markup attribute (i.e. role="button") on the root element which sets the type of the component. For a full listing of roles see [Roles Categorization](https://www.w3.org/TR/wai-aria-1.1/#roles_categorization).
+        ```
+        <div tabindex="0" role="button" id="action">Print Page</div>
+        ```
+        * HTML5 landmark region - A role can be achieved automatically without using the role attribute by using HTML5 section elements (such as nav, form). The assistive technology understands how to present this correctly to the user.
+        ```
+        <nav>
+            <ol>
+                <li>
+                    <a>Authoring Practice</a>
+                </li>
+                <li>
+                    <a>Cats</a>
+                </li>
+            </ol>
+        </nav>
+        ```
+    2) **WAI-ARIA state & property**: Aria-prefixed markup attribute which sets the state of a component (i.e. "aria-checked" for a checkbox) or a property of it (i.e. aria-label). Note that the difference between a state and property is semantic when considering the code implementation - states usually change a lot while properties do not. For a full listing see [States & Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
```

Comment:
suggestion:
aria-state is always an Internal Implementation Attribute ( = we never allow user to directly specify aria-state attributes, but rather deduct it from the owner component's state and props)